build/build-bundle.js

```diff
@@ -148,10 +148,11 @@ async function minifyScript(filePath) {
       comments: /^!/,
       max_line_len: 1000,
     },
-    // The config relies on class names for gatherers.
-    keep_classnames: true,
-    // Runtime.evaluate errors if function names are elided.
-    keep_fnames: true,
+    // Class and function names are needed for things like loading gatherers by
+    // name and accessing functions by name in Runtime.evaluate. Instead, just
+    // rely on whitespace elimination for the large savings.
+    compress: false,
+    mangle: false,
```

Comment:
there are a few permutations that work here, but I'm not sure if it matters (these just affect identifiers, so fewer bytes but parsers blaze through them so not a big deal if you aren't downloading our bundle).

The biggest savings would be something like:

```suggestion
    // The config relies on class names for gatherers.
    keep_classnames: true,
    // Runtime.evaluate errors if function names are elided.
    keep_fnames: true,
    mangle: {
        reserved: Object.keys(pageFunctions),
      },
```

which is essentially no change in size over the current bundle